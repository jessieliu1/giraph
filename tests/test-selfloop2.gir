int main() {
	wegraph<int> g = [A:3 -{5}- A];
	for_node(n : g) {
		print(n.data());
	}
	for_edge(e : g) {
		print(e.from().data());
		print(e.to().data());
		print(e.weight());
	}

	wedigraph<int> g2 = [B:4 -{6}-> B];
	for_node(n : g2) {
		print(n.data());
	}
	for_edge(e : g2) {
		print(e.from().data());
		print(e.to().data());
		print(e.weight());
	}
	return 0;
}