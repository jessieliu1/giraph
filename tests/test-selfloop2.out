3
3
3
5
4
4
4
6